packages/gatsby-link/README.md

```diff
@@ -31,6 +32,27 @@ render () {
       activeStyle={{
         color: 'red'
       }}
+    >
+    Another page
+    </Link>
   </div>
 }
 ```
+
+In typescript:
```

Comment:
This Typescript example is superfluous. It's 95% identical to the JS one and anyone writing Typescript would know how to translate the JS example to Typescript.

I can see value in maintaining code samples in different languages in the future but if we do, we'd want to put them in a toggleable interface so it's hiding the languages you're not interested in.